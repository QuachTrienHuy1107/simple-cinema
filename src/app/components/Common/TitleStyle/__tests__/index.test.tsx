import * as React from "react";
import { render } from "@testing-library/react";

import { TitleStyle } from "..";

jest.mock("react-i18next", () => ({
    useTranslation: () => {
        return {
            t: str => str,
            i18n: {
                changeLanguage: () => new Promise(() => {}),
            },
        };
    },
}));

describe("<TitleStyle  />", () => {
    it("should match snapshot", () => {
        const loadingIndicator = render(<TitleStyle />);
        expect(loadingIndicator.container.firstChild).toMatchSnapshot();
    });
});
