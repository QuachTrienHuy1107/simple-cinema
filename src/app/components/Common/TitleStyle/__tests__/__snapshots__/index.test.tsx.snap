// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TitleStyle  /> should match snapshot 1`] = `
.c0 {
  color: #000;
  font-weight: 500;
}

.c0:hover h3 {
  -webkit-transition: all 0.3s;
  transition: all 0.3s;
  color: #fb4226;
}

<a
  class="c0"
>
  <h3 />
</a>
`;
